transport_versioned_docs/version-1.0.0/workflow-client-server-secure.md

```diff
@@ -0,0 +1,147 @@
+---
+id: jobs
+title: Create jobified client and server
+---
+The unity transport protocol can be configure to encrypt the connection between the server and the client while ensuring the server's/client's authenticity.
+
+## Server authentication
+
+### High level authentication process
+In this configuration, the server will provide a certificate to the client (`rsa`)
+The client will then validate the provided certificate against the one its root certificate (`pem`) and will subsequently validate the server identity.
+Once its identity confirmed, the server will then use the private key (`rsaKey`) to encrypt the messages targeting the client.
+
+### Requirements
+To use the client server secure workflow, you need a valid certificate and the root certificate it has been generated from. You also need the private key that has been used to create the certificate.
+* Samples root certificate, private key and certificate are provided with the sample project. Please do not use them for production. 
+* It is of the upmost importance for you to have your own certificates. The procedure to generate them using OpenSSL is detailed hereafter. 
+
+### Generating the required keys and certificates with OpenSSL
+
+It is assumed to you have openSSL installed on your machine.
+
+#### Generate the Certification Authority root certificate. 
+First thing first is to generate a private key for that will help to generate the certification authority root certificate. 
+```shell
+openssl genrsa -des3 -out clientPrivateKeyForRootCA.key 2048
```

Comment:
Can leave out `-des3` to not have to bother with a passphrase. If we care enough about good practices to set a passphrase, then should use something more up-to-date than DES3 (e.g. something like `-aes256`).

Reply:
Removed the des3.